plugins/repository-hdfs/build.gradle

```diff
@@ -229,6 +229,10 @@ if (fixtureSupported) {
   integTest.dependsOn hdfsFixture
   integTestHa.dependsOn haHdfsFixture
 
+  if (project.ext.has('inFipsJvm') && project.ext.inFipsJvm) {
```

Comment:
AFAIK the property won't exist at config time, right @mark-vieira using  `onlyIf` 
should work becuase the closure is ran at execution time.

Reply:
Correct. The preferred way to skip execution of a task when it is non-applicable is via `onlyIf`.

```groovy
integTest.onlyIf { inFipsJvm }
```